### ISSUE TITLE
Tests run should be queued

### ISSUE THREAD

**Issue by gerdint** [microsoft/vscode-java-test#780]
If one attempts to start a test while the test runner is already in the process of running tests a notice indicating that 
When requesting a test to be run there is a message saying "A test session is currently running. Please wait until it finishes." It would be more useful if the tests instead were queued and then run once the first tests are finished.

**Comment by jdneo** [comment]
_According to @aghasemi in https://github.com/microsoft/vscode-java-test/issues/873:_

```
Hi,
Is there any way to run a test when another is still being executed, somehow as two separate processes, without mixing the output?

Thanks
```

**Comment by jdneo** [comment]
Addressed in 0.31.0
